OpenTap.Plugins.PNAX/LMS/StoreSnp.cs

```diff
@@ -45,33 +49,52 @@ public class StoreSnp : TestStep
 
         public StoreSnp()
         {
-            Channel = 1;
-            mnum = 1;
+            channels = new List<int>() { 1 };
+            //mnum = 1;
             Ports = new List<int>() { 1, 2 };
-            filename = new MacroString(this) { Text = "CH1" };
+            filename = new MacroString(this) { Text = "<PartId>" };
```

Comment:
`<PartId>` is provided by the **Track Work Order** plugin. 
* What is the behavior if that plugin isn't present? Do we need to handle that for the general market use (_e.g. what happens if someone who doesn't have **Track Work Order** runs a step that relies on this setting_)?